src/webextension/telemetry.js

```diff
@@ -0,0 +1,9 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Is this file still used?

Reply:
Not currently, but it will be for any telemetry events recorded in content pages (e.g. the management UI).